src/RequestProxyFactory.php

```diff
@@ -57,23 +62,22 @@ public function createProxyRequest(RequestInterface $request): RequestInterface
             $request = $request->withoutAttribute('_encoder');
         }
 
-        // Strip any authentication headers from the request.
-        $request = $request->withoutHeader('Authorization');
+        // Strip all headers.
```

Comment:
Note: (I had to check) it doesnt appear Elasticsearch requires any headers for normal operation, so stripping headers is :chefs-kiss: